### PR TITLE
add support for oracle database

### PR DESCRIPTION
**Pull request by dbu**
as discussed in #399

fix #399

### CODE REVIEW

**Comment by dbu** [comment]
@jjwdesign can you confirm if this is all that is needed to make oracle supported out of the box?

File: README.md [hunk 1]
```diff
@@ -68,6 +69,11 @@ Database is created automatically if you specify driver and path ("pdo_sqlite",
 
 For further details, please see the [Doctrine configuration page](http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connection-details).
 
+### Oracle
+
+Note: A doctrine middleware is automatically added to the database connection to work around Oracle converting the lowercase
+table and field names to upper case in the responses.
```
Comment (dbu) [inline README.md]: todo: add disclaimer that we have no automatic tests for the oracle integration

**Comment by jjwdesign** [comment]
@dbu   Yes, that was the only issue we were seeing when using jackalope/jackalope-doctrine-dbal with Oracle Database 19c Enterprise Edition. Thank you for making this update!

File: README.md (hunk 1 above)
Comment (dbu) [reply README.md]: done

**Comment by dbu** [comment]
> @dbu Yes, that was the only issue we were seeing when using jackalope/jackalope-doctrine-dbal with Oracle Database 19c Enterprise Edition. Thank you for making this update!

awesome, thanks for your confirmation. i will tag 1.8.0 shortly. with that, you should no longer need custom code as the lowercase middleware will then automatically be added (i implemented it so that we only add it if it is not already in the list of middleware)

**Pull request merged**